ietf/doc/templatetags/ietf_filters.py

```diff
@@ -881,3 +881,19 @@ def badgeify(blob):
             )
 
     return text
+
+@register.filter
+def has_chat_logs(meeting):
```

Comment:
Should this get the same "Return true for interims" treatment?

Reply:
It's not clear that these should be filters - why aren't they methods on Meeting instead? (those would be visible to the templates). I think you were on the way to discovering that with the partial removal of the has_notes filter, and the redundant argument passed to the templates in the views later.

Reply:
Happy to make them methods on the meeting if you prefer. I didn't know templates could invoke methods directly :)

Reply:
> Should this get the same "Return true for interims" treatment?

I guess?  I changed it in the commit 466a6306a